StripeFinancialConnections/StripeFinancialConnections/Source/Helpers/UIColor+Extensions.swift

```diff
@@ -171,6 +171,10 @@ extension UIColor {
         return UIColor(red: 26 / 255.0, green: 197 / 255.0, blue: 155 / 255.0, alpha: 1)  // #1ac59b
     }
 
+    static var linkGreen900: UIColor {
+        return UIColor(red: 1 / 255.0, green: 30 / 255.0, blue: 15 / 255.0, alpha: 1)  // #011E0F
```

Comment:
[Source](https://www.figma.com/design/mhgvqbODPxfR4c0j2u3ySn/%E2%9D%87%EF%B8%8F-Link-Design-System-2.0?node-id=385-57124&t=YvBouyaJvmo0BLZ6-4)

Reply:
It's pretty subtle, but this is the colour used for the web link logo fill as well 
<img width="1421" alt="image" src="https://github.com/user-attachments/assets/25217a23-5b12-45a3-a83c-41b4ec5293f8">